src/masonite/commands/presets/React.py

```diff
@@ -48,5 +48,6 @@ def update_component(self):
         shutil.copyfile(os.path.dirname(__file__) + '/react-stubs/Example.js', 'resources/js/components/Example.js')
 
     def update_bootstrapping(self):
-        """Copies template app.js file into application"""
+        """Copies template app.js and bootstrap.js into application"""
         shutil.copyfile(os.path.dirname(__file__) + '/react-stubs/app.js', 'resources/js/app.js')
+        shutil.copyfile(os.path.dirname(__file__) + '/remove-stubs/bootstrap.js', 'resources/js/bootstrap.js')
```

Comment:
I think it would be better to either move bootstrap.js to a generic location OR copy it into each stub directory where it is needed.

Reply:
I moved `bootstrap.js` to a generic location.

Reply:
Thanks